src/main/java/com/superdevs/HealthOMeter/dto/BioMetricDataDto.java

```diff
@@ -0,0 +1,87 @@
+package com.superdevs.HealthOMeter.dto;
+
+import com.superdevs.HealthOMeter.entity.Contact;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+
+public class BioMetricDataDto {
+
+    private long id;
+    private BigDecimal growth;
+    private BigDecimal weight;
+    private BigDecimal waistCirc;
+    private BigDecimal hipCirc;
+    private LocalDateTime created;
+    private Contact contact;
+
+    public BioMetricDataDto() {
+    }
+
+    public BioMetricDataDto(long id, BigDecimal growth, BigDecimal weight, BigDecimal waistCirc,
+                         BigDecimal hipCirc, Contact contact) {
+        this.id = id;
```

Comment:
juz dalej o tym id nie pisze

Reply:
Fixing it right now, great suggestion, make sense.